ibm/service/kms/data_source_ibm_kms_key_test.go

```diff
@@ -32,6 +32,27 @@ func TestAccIBMKMSKeyDataSource_basic(t *testing.T) {
 	})
 }
 
+func TestAccIBMKMSKeyDataSource_description(t *testing.T) {
+	instanceName := fmt.Sprintf("kms_%d", acctest.RandIntRange(10, 100))
+	// bucketName := fmt.Sprintf("bucket", acctest.RandIntRange(10, 100))
```

Comment:
Is there a reason for the commented out `bucketName`s?
```suggestion
```

Reply:
Oh oops that was supposed to be removed